param/tasks.yaml

```diff
@@ -9,7 +9,11 @@ takeoff_velocity        : 0.3
 takeoff_acceleration    : 0.3
 # Height at which to consider takeoff finished in meters
 # Should be higher than min_maneuver_height
-takeoff_complete_height: 0.5
+takeoff_complete_height: 0.8
+# Deviance from takeoff height from which to enter stabilize mode
+takeoff_complete_height_tolerance: 0.15
+# Length of time to stabilize after takeoff
+takeoff_stabilize_delay: 1.5
```

Comment:
Units please for these 3 params

Reply:
The response from both of our beloved leads is that we always use mks units, and my response to this is that there should just be two words at the top - "mks units". It's a standard across every single one of our yaml files (as far as I've seen) but it isn't a big deal to have it on top of each file

Reply:
We are following REP103 for units so I don't think they need to be specified.

http://www.ros.org/reps/rep-0103.html#units